tests/client/test_prefect_client.py

```diff
@@ -1907,19 +1907,6 @@ async def test_get_runs_from_queue_respects_limit(self, prefect_client, deployme
         assert len(output) == 10
         assert {o.id for o in output} == {r.id for r in runs}
 
-    async def test_get_runs_from_queue_updates_status(
-        self, prefect_client: PrefectClient
-    ):
-        queue = await prefect_client.create_work_queue(name="foo")
-        assert queue.status == "NOT_READY"
-
-        # Trigger an operation that would update the queues last_polled status
-        await prefect_client.get_runs_in_work_queue(queue.id, limit=1)
-
-        # Verify that the polling results in a READY status
-        lookup = await prefect_client.read_work_queue(queue.id)
-        assert lookup.status == "READY"
-
```

Comment:
This test is flaky and duplicative of the server side test suite, so I removed it.